src/test/resources/debug-config/application.properties

```diff
@@ -4,6 +4,10 @@ fcrepo.url=http://deasy.dans.knaw.nl:8080/fedora
 fcrepo.user=fedoraAdmin
 fcrepo.password=fedoraAdmin
 
+fsrdb.db-connection-url=jdbc:postgresql://deasy.dans.knaw.nl:5432/easy_db
+fsrdb.db-connection-username=easy_webui
+fsrdb.db-connection-password=changeme
```

Comment:
Don't forget easy-dtap.

The other passwords in the debug config use the deasy value. The dist config uses changeme. Look for the correct strategy in other projects.